susemanager/bin/server-migrator.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+spacewalk-service stop
+cp -a /var/lib/cobbler /var/lib/cobbler.old
+cp -a /var/lib/rhn/kickstarts /var/lib/rhn/kickstarts.old
+rm /var/lib/cobbler/snippets/spacewalk
+mv /etc/zypp/repos.d /etc/zypp/repos.d.old
+mkdir /etc/zypp/repos.d
+zypper ar -n "Main Repository" http://download.opensuse.org/distribution/leap/15.1/repo/oss repo-oss
+zypper ar -n "Main Update Repository" http://download.opensuse.org/update/leap/15.1/oss repo-update
+zypper ar -n "Non-OSS Repository" http://download.opensuse.org/distribution/leap/15.1/repo/non-oss repo-non-oss
+zypper ar -n "Update Repository (Non-Oss)" http://download.opensuse.org/update/leap/15.1/non-oss/ repo-update-non-oss
+zypper ar -n "Uyuni Server 4.0.2" https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master/images-openSUSE_Leap_15.1/repo/Uyuni-Server-4.0-POOL-x86_64-Media1/ uyuni-server-4.0.2
```

Comment:
As I don't think we want to change this with every version, I'd avoid using `4.0.2`. Is that possible or do we then have a conflict with an existing repository?

Reply:
You mean the repository label "uyuni-server-4.0.2"? Of course we can change this to whatever you like.

Reply:
And the repository name at the end of the line.

Maybe we can set both to `uyuni-server-4.0`?

Reply:
I have used the term "latest" now else we again have version numbers :)